tooling/camel-karaf-test-feature-archetype/src/main/resources/archetype-resources/src/main/java/test/Camel__featureName__RouteSupplier.java

```diff
@@ -27,7 +27,7 @@
 @Component(
         name = "karaf-camel-${featureNameLower}-test",
         immediate = true,
-        service = Camel${featureName}RouteSupplier.class
+        service = CamelRouteSupplier.class
```

Comment:
Could you please move this change to the other PR (https://github.com/apache/camel-karaf/pull/509) which is more related

Reply:
yes, will do it 👍